pyccel/parser/semantic.py

```diff
@@ -3496,6 +3496,17 @@ def _visit_FunctionDef(self, expr):
 
     def _visit_PythonPrint(self, expr):
         args = [self._visit(i) for i in expr.expr]
+        for i, arg in enumerate(args):
+            if hasattr(arg.value, 'rank') and arg.value.rank and not isinstance(arg.value, (Variable, PythonTuple, IndexedElement)):
```

Comment:
Why not use the default argument for `getattr` to make this simpler:
```suggestion
            if getattr(arg.value, 'rank', 0) and not isinstance(arg.value, (Variable, PythonTuple, IndexedElement)):
```